src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 10px;
```

Comment:
[Check list №11](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) [STYLES] - Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover.

So it's means that padding should be equal or bigger than value that you use to move calendar days. 
` transform: translate(0, -20px);`
Also you can move this value to variables.